Public/Src/Cache/ContentStore/Distributed/NuCache/BinManager.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.ContractsLight;
+using BuildXL.Cache.ContentStore.Hashing;
+using BuildXL.Cache.ContentStore.Utils;
+
+namespace BuildXL.Cache.ContentStore.Distributed.NuCache
+{
+    internal class BinManager
+    {
+        private const int MaxBins = 64 * 1024;

```

Comment:
>64 * 1024; [](start = 36, length = 10)

ushort.MaxValue

Reply:
Not the same. ushort.MaxValue == (64 * 1024) - 1

---
In reply to: [347646590](https://github.com/microsoft/BuildXL/pull/1220#discussion_r347646590) [](ancestors = 347646590)